astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -31,6 +38,23 @@
 from astronomer.providers.utils.typing_compat import Context
 
 
+def _check_queries_finish(conn: SnowflakeConnection, query_ids: list[str]) -> bool:
+    with closing(conn) as conn:
+        for query_id in query_ids:
+            status = conn.get_query_status(query_id)
+
+            if status == QueryStatus.ABORTING:
+                raise AirflowException(f"ABORTING {ABORTING_MESSAGE}")
+            elif status == QueryStatus.FAILED_WITH_ERROR:
+                raise AirflowException(f"FAILED_WITH_ERROR {FAILED_WITH_ERROR_MESSAGE}")
+            elif status == QueryStatus.SUCCESS:
+                pass
+            else:
```

Comment:
What is other query statuses? it would be good to log in with warning before returning False. WDYT?

Reply:
I think `warning` might not be the best solution here. Because even if it returns `false`, it just means we're going to defer the task. But for better readability, I'll make the following changes 

```suggestion
            elif (
                    QueryStatus.RUNNING,
                    QueryStatus.QUEUED,
                    QueryStatus.DISCONNECTED,
                    QueryStatus.RESUMING_WAREHOUSE,
                    QueryStatus.BLOCKED,
                    QueryStatus.NO_DATA,
            ):
                return False
            else:
                raise ValueErorr(f"Unexpected QueryStatus: str(status)")
```